src/abi/struct.ts

```diff
@@ -103,6 +103,51 @@ export const StructFunctions: Map<string, AbiFunction> = new Map([
             },
         },
     ],
+    [
+        "toSlice",
+        {
+            name: "toSlice",
+            resolve: (ctx, args, ref) => {
+                if (args.length !== 1) {
+                    throwCompilationError(
+                        "toSlice() expects no arguments",
+                        ref,
+                    );
+                }
+                const arg = args[0]!;
+                if (arg.kind !== "ref") {
+                    throwCompilationError(
+                        "toSlice() is implemented only a struct type",
```

Comment:
We also have `toSlice()` for string builders: https://github.com/tact-lang/tact/blob/5bec4864deccf8a3ee694914f3be0a815818488e/stdlib/std/text.tact#L30

Reply:
Then should I also change the `toCell() is implemented only a struct type` error message for `toCell()` as there's also that function in stdlib for `StringBuilder`?

Reply:
Sure, let's add it here too

Reply:
Maybe it'll be better to change these error messages to something like `toSlice() is not implemented for type 'type'`?

Reply:
Sure, sounds good